keps/sig-auth/3926-handling-undecryptable-resources/README.md

```diff
@@ -405,7 +406,61 @@ deletions should therefore have their own extra admission.
 
 The unconditional deletion admission:
 1. checks if a "delete" request contains the `IgnoreStoreReadErrorWithClusterBreakingPotential` option
-2. if it does, it checks the RBAC of the request's user for the `delete-ignore-read-errors` verb of the given resource
+2. if it does, it checks the RBAC of the request's user for the `unsafe-delete-ignore-read-errors` verb of the given resource
+
+
+
+#### Propagating with WATCH
+When a corrupt object is deleted with a `unsafe-delete-ignore-read-errors`, a
+watcher of this resource will not be able to transform its old data from the
+storage, or decode it into an object [1]. This will cause the watch to throw
+an `watch.ERROR` event.
+A client backed up an informer already has the object in its cache, since the
+client never receives a `watch.DELETED` event the object remains in the lsiter
+cache. This creates an inconsistency - retrieving the object from the cache
+yields the object, but if we get it from the storage we see a `corrupt object`
+error.
```

Comment:
Based on the proposed solution, this means that all existing clients will remain in a corrupted state indefinitely without the ability to recover.  To have a safe rollout for our own clients (not even considering external clients yet), this means we need to promote handling for this special error to locked-to-true status until every supported kubelet (n-3) has the value locked-to-true before we can start enabling the server-side capability.  Otherwise we can end up with unrecoverable corruption, correct?